packages/twenty-front/src/testing/mock-data/generated/mock-metadata-query-result.ts

```diff
@@ -9080,28 +9080,6 @@ export const mockedStandardObjectMetadataQueryResult: ObjectMetadataItemsQuery =
                     relationDefinition: null,
                   },
                 },
-                {
```

Comment:
I guess we should also add it on the calendarChannelMessageAssociation but it will automatically be done in the next mock refresh, should not have any impact on the tests